test/libsolidity/SemanticTest.cpp

```diff
@@ -46,77 +46,102 @@ SemanticTest::SemanticTest(string const& _filename, string const& _ipcPath, lang
 	m_source = parseSourceAndSettings(file);
 	if (m_settings.count("compileViaYul"))
 	{
-		m_validatedSettings["compileViaYul"] = m_settings["compileViaYul"];
-		m_compileViaYul = true;
-		m_settings.erase("compileViaYul");
```

Comment:
Won't removing this line have any effects?

Reply:
That wasn't actually intentional... but apparently it *doesn't* have any effects :-D. I'll add it back anyways.

Reply:
Added it back.